### PR TITLE
Extract UDP backend into the pydevialet_expert_nonpro library

### PR DESCRIPTION
**Pull request by brew-your-own**
Turn the reverse-engineered UDP communication code into a standalone, dependency-free, PyPI-installable Python library so it can be reused by any application instead of being embedded in the Kivy GUI.

- Add the src-layout package src/pydevialet_expert_nonpro/ exposing DeviMoteBackEnd (alias DevialetExpertNonPro). Pure standard library.
- Replace the hand-rolled CRC-16/CCITT-FALSE with binascii.crc_hqx, verified equivalent via the canonical 0x29B1 check value.
- Add pyproject.toml (setuptools, src layout); the GUI ships via a gui extra and is intentionally excluded from the wheel.
- Add unit tests for the CRC, dB volume encoding, and status-packet decoding; none require an amplifier.
- Move the Kivy GUI to gui/devimote.py + gui/devimote.kv as a standalone script that imports the library instead of embedding the backend.
- Update CI to install the package and run pytest + pylint; switch the REUSE workflow to the official action.

### CODE REVIEW

**Comment by gnulabis** [comment]
Thanks @brew-your-own , this looks great! I've tried it and it works fine.

Where is the CLI?

**Comment by brew-your-own** [comment]
ah good point, I did not include the CLI, I made the minimal change to separate / make it a library
I can add the CLI in a different PR once this one is merged.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# SPDX-License-Identifier: CC0-1.0
+
+[project]
+name = "pydevialet-expert-nonpro"
+version = "0.1.0"
+description = "Unofficial library for controlling Devialet Expert (non-Pro, pre-Core-Infinity) amplifiers over UDP"
+requires-python = ">=3.9"
+license = "GPL-3.0-or-later"
+authors = [
+    { name = "Dimitris Lampridis" },
```
Comment (gnulabis) [inline pyproject.toml]:
```suggestion
    { name = "Dimitris Lampridis", email = "dlampridis@gmail.com" },
```

File: README.md [hunk 2]
```diff
@@ -5,9 +5,65 @@ SPDX-License-Identifier: CC0-1.0
...
+
+The Kivy GUI is a standalone script. Install the project with the `gui` extra and run it:
+
+```bash
+pip install -e ".[gui]"
+python gui/devimote.py
+```
+
 ![](doc/images/devimote_demo.gif)
+
```
Comment (gnulabis) [inline README.md:61]:
```suggestion

> [!NOTE]  
> the Kivy 2.3.1 dependency [does not install with Python 3.14](https://github.com/kivy/kivy/issues/9225). If you want to test/use the GUI, you must use Python 3.13 or earlier.

```
